omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -248,6 +248,11 @@ Start the PostgreSQL server.
     $ brew services start postgresql
     $ pg_ctl -D /usr/local/var/postgres/ -l /usr/local/var/postgres/server.log start
 
+You will need to install ``brew services`` first:
```

Comment:
Shouldn't this section go above `Start the PostgreSQL server`, since it's a prerequisite for the `brew services start`?